paf-mvp-frontend/src/lib/tsconfig.json

```diff
@@ -1,15 +0,0 @@
-{
-  "extends": "../../../tsconfig.json",
-  "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "declaration": true,
-    "declarationDir": "",
-    "types": [
-      "node"
-    ]
-  },
-  "include": [
-    "./paf-lib.ts"
```

Comment:
It seems that paf-lib.ts grows (> 700 lines). At some point, it will be relevant to split it in different files (e.g paf-lib-id-prefs.ts, paf-lib-seed-audit.ts). Does it make sense to set up a rule right now to ease this split ?

ping @OlivierChirouze

Reply:
Of course, we discussed it with Olivier today morning